### PR TITLE
Set favicon size to tabbar size.

### LINKED ISSUE #119: Adjust favicons to text size
Favicons in the tabs should be sized according to the font size, not hardcoded to 12x12

### PR DESCRIPTION
**Pull request by Carpetsmoker**
Fixes #119. Also see #754

### CODE REVIEW

**Comment by The-Compiler** [comment]
I'm a bit confused - how does this relate to #754? Isn't the same functionality present there already?

**Comment by Carpetsmoker** [comment]
#754 adds a new setting to control the favicon size.

This gets the size of the statusbar, and sets the size of the favicon to that (minus some padding). Since the tabbar size is implied from the font size, this does that you added in last comment in #754 :

> For the record: We discussed this in IRC and agreed it'd be better to add an option to set the tabbar size - then the favicon size adjusts to that (if favicon > tabbar), or gets drawn smaller (if favicon < tabbar).

File: qutebrowser/browser/network/qutescheme.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def qute_help(win_id, request):
     """Handler for qute:help. Return HTML content as bytes."""
     try:
         utils.read_file('html/doc/index.html')
-    except FileNotFoundError:
+    except (FileNotFoundError, NotADirectoryError, OSError, BlockingIOError):
```
Comment (The-Compiler) [inline qutebrowser/browser/network/qutescheme.py]: This seems like an unrelated change from #763 - can you remove it from the PR please? :smile:

Comment (Carpetsmoker) [inline qutebrowser/browser/network/qutescheme.py]: Whoops, fixed!
